lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -1597,4 +1644,142 @@ describe('ContentMetadataComponent', () => {
             expect(customComponent).toBeDefined();
         });
     });
+
+    describe('Predictions', () => {
+        const getMockPrediction = (reviewStatus: ReviewStatus): Prediction => ({
+            confidenceLevel: 0.9,
+            predictionDateTime: new Date(2024, 1, 1),
+            modelId: 'test-model-id',
+            property: 'test:test',
+            id: 'test-prediction-id',
+            previousValue: 'previous value',
+            predictionValue: 'new value',
+            updateType: 'AUTOCORRECT',
+            reviewStatus: reviewStatus
+        });
+
+        const getMockPredictionPaging = (predictions: Prediction[]): PredictionPaging => ({
+            list: {
+                entries: predictions.map((prediction) => ({ entry: prediction }))
+            }
+        });
+
+        let getPredictionsSpy: jasmine.Spy;
+
+        beforeEach(() => {
+            component.node = node;
+            component.displayPredictions = true;
+            getPredictionsSpy = spyOn(predictionService, 'getPredictions').and.returnValue(
+                of(getMockPredictionPaging([getMockPrediction(ReviewStatus.UNREVIEWED)]))
+            );
+            fixture.detectChanges();
```

Comment:
I guess if you remove this line then you won't have to call component.ngOnInit() in every test case. You'd still have to call fixture.detectChanges() everytime though. I'm not sure if calling component.ngOnInit() reruns the entire component initialisation lifecycle or not, but if it does, then this could be a performance improvement, since we won't be running the component init lifecycle twice.

Reply:
unfortunately it didn't help